Makefile

```
@@ -99,3 +99,14 @@ build-image-collector:
 		-t ${REGISTRY}/${COLLECTOR_IMAGE_NAME}:${COLLECTOR_IMAGE_TAG} \
 		-t ${REGISTRY}/${COLLECTOR_IMAGE_NAME}:${COLLECTOR_VERSION} \
 		-f Dockerfile .
+
+build-and-deploy-image-collector:
```

```suggestion
build-and-deploy-image-collector-dev:
```

Maybe something like this?